__tests__/app.test.js

```diff
@@ -1,58 +1,49 @@
 require('dotenv').config();
 
-const { execSync } = require('child_process');
+// const { execSync } = require('child_process');
 
 const fakeRequest = require('supertest');
 const app = require('../lib/app');
-const client = require('../lib/client');
+// const client = require('../lib/client');
 
 describe('app routes', () => {
   describe('routes', () => {
-    let token;
-  
-    beforeAll(async () => {
-      execSync('npm run setup-db');
-  
-      await client.connect();
-      const signInData = await fakeRequest(app)
-        .post('/auth/signup')
-        .send({
-          email: 'jon@user.com',
-          password: '1234'
-        });
-      
-      token = signInData.body.token; // eslint-disable-line
-    }, 10000);
-  
-    afterAll(done => {
-      return client.end(done);
-    });
+    test('returns location', async() => {
+
+      const expectation = {
+        formatted_query: 'Albuquerque, Bernalillo County, New Mexico, USA',
+        latitude: '35.0841034',
+        longitude: '-106.6509851'
+      };
+      const data = await fakeRequest(app)
+        .get('/location?search=Albuquerque')
+        .expect('Content-Type', /json/)
+        .expect(200);
 
-    test('returns animals', async() => {
+      expect(data.body).toEqual(expectation);
+    });
+    test('returns weather', async() => {
 
       const expectation = [
```

Comment:
It's really good to test for the shape of the results like this, but consider checking the Jest documentation to see if there's a way to make this less repetitive. (Maybe you can test for an array of objects that have a certain shape?)

Reply:
(After seeing the same repetitive expectation array in lab after lab, I realized that's how Dani wrote it in the demo; it makes sense now! It's still worth noting that it can probably be made DRYer.)